### PR TITLE
Identify and label a bunch of game.c stuff

### PR DESCRIPTION
**Pull request by FazanaJ**
Also fixed some inaccurate labelling from past me.

### CODE REVIEW

File: src/objects.c [hunk 1]
```diff
@@ -2263,11 +2266,11 @@ s8 func_8001A7D8(unk8001A7D8_arg0 *arg0) {
     }
     D_8011ADC4 = 0;
     if (!(arg0->unk4[arg0->unk49] & 2)) {
-        if (D_8011AEF5 == 0) {
+        if (gIsTimeTrial == FALSE) {
```
Comment (DavidSM64) [inline src/objects.c:2269]:
```suggestion
        if (!gIsTimeTrial) {
```

File: src/objects.c [hunk 2]
```diff
@@ -2263,11 +2266,11 @@ s8 func_8001A7D8(unk8001A7D8_arg0 *arg0) {
...
             D_8011ADC4 = 1;
             arg0->unk4[arg0->unk49] |= 2;
         }
-    } else if (gIsSilverCoinRace && racer->silverCoinCount >= 8 && D_8011AEF5 == 0) {
+    } else if (gIsSilverCoinRace && racer->silverCoinCount >= 8 && gIsTimeTrial == FALSE) {
```
Comment (DavidSM64) [inline src/objects.c:2273]:
```suggestion
    } else if (gIsSilverCoinRace && racer->silverCoinCount >= 8 && !gIsTimeTrial) {
```

File: src/racer.c [hunk 3]
```diff
@@ -3110,7 +3110,7 @@ void func_80050A28(Object *obj, Object_Racer *racer, s32 updateRate, f32 updateR
     // Degrade lateral velocity
     if (gCurrentPlayerIndex != PLAYER_COMPUTER) {
         if (!(racer->velocity > -2.0) && racer->drift_direction == 0 && racer->raceFinished == FALSE) {
```
Comment (DavidSM64) [inline src/racer.c:3112]:
```suggestion
        if (!(racer->velocity > -2.0) && racer->drift_direction == 0 && !racer->raceFinished) {
```

**Review by DavidSM64 — APPROVED** [review]
Very minor nitpick. I'd prefer `!var` over `var == FALSE`

File: src/game_ui.c [hunk 4]
```diff
@@ -290,8 +290,8 @@ s32 D_80127190;
 s32 D_80127194;
 s32 D_80127198[6];
 
-// ????
-u8 gGfxTaskYieldData[0xA00];
+// This should really be in
```
Comment (Ryan-Myers) [inline src/game_ui.c:293]: Where!?!? 😉

**Review by Ryan-Myers — APPROVED** [review]
LGTM

File: src/game_ui.c (hunk 4 above)
Comment (FazanaJ) [reply src/game_ui.c]: danmbit

**Pull request merged**